CONTRIBUTING.md

```diff
@@ -11,8 +11,8 @@ All contributions should be sent as
 ### License
 
 All software (code, associated documentation, support files, etc) in the
-Symbiflow repositories are licensed under the very permissive
-[ISC Licence](https://opensource.org/licenses/ISC). A copy can be found in the [`LICENSE`](LICENSE) file.
+F4PGA repositories are licensed under the very permissive
+[Apache 2.0 Licence](https://opensource.org/licenses/Apache-2.0). A copy can be found in the [`LICENSE`](LICENSE) file.
```

Comment:
The current LICENSE file corresponds to ISC. Please update it in this PR.

Reply:
The repositories that this file syncs with, listed in `.github/sync.yml`, all have the Apache 2.0 License, which is why I have updated it here, since this is the file that will be copied over to those target repositories. I understand the confusion though, since the `symbiflow-common-config` repository does not have this license. Maybe the best solution would be to make another `CONTRIBUTING.md` file for this repository, then rename the `CONTRIBUTING.md` file that will be used to sync to other repositories. Let me know what you think. @umarcor

Reply:
Actually, I meant to change https://github.com/SymbiFlow/symbiflow-common-config/blob/main/LICENSE, because the license of `symbiflow-common-config` does also need to be Apache 2.0.

I see the misunderstanding you are explaining. I do think it would be sensible to make it explicit which are the "assets" and which are the sources corresponding to the `symbiflow-common-config` repository itself. Therefore, I would suggest to go ahead and rename the current `CONTRIBUTING.md` to something else (say `CONTRIBUTING.ref.md`) or move it into the assets subdir. However, I don't know if the Action being used ATM does support the files having a different name or being in a different location. A different (or custom, https://github.com/SymbiFlow/actions) Action might be required in order to support such use case.